tools/templates/lint-and-validate-values.yaml

```diff
@@ -176,6 +176,10 @@ singleuser:
   extraEnv: {}
   lifecycleHooks:
   initContainers:
+  extraContainers:
+  - name: mock-extra-container-name
+    image: mock-extra-container-image-name
+    imagePullPolicy: IfNotPresent
```

Comment:
Can you copy and add it for the initContainers as well, and indent the name/image/imagePullPolicy two spaces?

Thank you for hanging in there!

Reply:
Sorry the indent is my mistake, i'll fix right now and add initContainers as well.

Reply:
Wooo this is awesome you are closing two issues and making a PR that I'll get very happy about merging thank you!